dockerfiles/Dockerfile-delegation-backend-toolchain

```diff
@@ -0,0 +1,4 @@
+FROM golang:1.16
+
+COPY result/headers /headers
```

Comment:
At some stage this should really be handled by some more metadata/go packaging in the c reference signer package so that you can put it in go.mod

Reply:
As I said, we'll rework it when migrate to rust implementation